lib/auth_mvp_web/templates/auth_url/index.html.eex

```diff
@@ -0,0 +1,10 @@
+<section class="phx-hero">
+  <h1>Welcome to DWYL Authentication App!</h1>
+  <p>To get started, login to your Google or Github ccount: <p>
+  <a href="<%= @oauth_google_url %>">
+    <img src="https://i.imgur.com/Kagbzkq.png" alt="Sign in with Google" />
```

Comment:
@SimonLab using the `<img>` button is fine for now. 👍 
At some point we can convert them to SVG for consistency. see: https://github.com/dwyl/auth/issues/44